postgresql/spoon.me

```diff
@@ -0,0 +1,54 @@
+##
+# Licensed under the Apache License, Version 2.0
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Setup meta
+meta title="postgresql"
+meta namespace="postgresql" 
+meta name="postgresql"
+
+#Basic dependencies and tools
+from clean,microsoft/vcredist:2008
```

Comment:
No...postgre sql explicitily need it. It states in the documentation that you should install it when using the Zip installer!!